### DISCUSSION TITLE
Using Raspberry PI 3B (not plus)

### DISCUSSION THREAD

**Discussion by odflor** [frntc/Sidekick64#85] [General]
Hi!

I'm using the SideKick64 Project with my Raspberry PI 3B (not plus) SOC.

Also I'm using it with my c128 NTSC "Slim".

Normally I need to power my RbPI with its own power source about of 5A capability.

Even so, I usually experience some gliches.

Is important that I need to use a RbPI 3B+(plus) instead of my RbPI3B (not plus)?

Best Regards

**Comment by frntc** [comment]
see https://github.com/frntc/RAD/discussions/24
